modules/oracle-xe/src/test/java/org/rnorth/testcontainers/containers/JDBCDriverTest.java

```diff
@@ -0,0 +1,43 @@
+package org.rnorth.testcontainers.containers;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.apache.commons.dbutils.QueryRunner;
+import org.junit.Test;
+
+import java.sql.SQLException;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class JDBCDriverTest {
```

Comment:
Can you name to the effect that this is the OracleJDBCDriverTest?

Also bonus points if you fancy pulling up the performSimpleTest and getDataSource methods into a shared TestUtils class that sits in the JDBC module ;)